src/main/scala/io/iohk/ethereum/domain/Address.scala

```diff
@@ -33,6 +35,11 @@ object Address {
     require(bytes.length <= Length, s"Invalid address: $hexString")
     Address(bytes)
   }
+
+  def apply(keyPair: AsymmetricCipherKeyPair): Address = {
+    val pub = crypto.pubKeyFromKeyPair(keyPair)
+    Address(crypto.kec256(pub).drop(FirstByteOfAddress))
```

Comment:
minor:  `.drop(FirstByteOfAddress)` is probably not necessary , as `Address` bytesting constructor takes only last 20byte also `FirstByteOfAddress` is weird name for something which drops 12 bytes